### PR TITLE
Replace old log_*() calls with Google glog

### PR DESCRIPTION
**Pull request by sangjinhan**
(along with other style fixes)

### CODE REVIEW

File: core/utils/format.cc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  va_end(ap);
+  return s;
+}
+
+std::string FormatVarg(const char *fmt, va_list ap) {
+  const int init_bufsize = 128;
+  std::unique_ptr<char[]> buf;
+
+  buf.reset(new char[init_bufsize]);
+  int ret = vsnprintf(&buf[0], init_bufsize, fmt, ap);
```
Comment (barath) [inline core/utils/format.cc]: Just curious -- why not vasprintf()?  Should simplify the buffer sizing aspect.

Comment (barath) [inline core/utils/format.cc]: (It's true that vasprintf() is a GNU/Linux/BSD function, not standard C, so that could be a reason not to use it.)

Comment (sangjinhan) [inline core/utils/format.cc]: I had no particular reason (just copy-and-pasted Chang's one from message.h/cc). I updated the code with vasprintf().

**Comment by melvinw** [comment]
Looks good. +1

**Pull request merged**